recipe/meta.yaml

```diff
@@ -7,51 +7,92 @@ package:
 source:
   fn: pymongo-{{ version }}.tar.gz
   url: https://github.com/mongodb/mongo-python-driver/archive/{{ version }}.tar.gz
-  sha256: c3b1be222869a35fee67714b754ebe8c5e83553b34ec60a6e8870ba0ea4f4496
+  sha256: 3639d459b06257b529c5675bb68e0f89d4a4d77fe5f76c01107202c3ba52d4cb
+  patches:
+    - patches/0001-generate-normalized-platform-tag.patch
 
 build:
   number: 0
-  skip: true  # [py<37]
-  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -v
+  # osx-arm64: "pymongo 4.8.0 is not supported on this platform"
+  skip: true  # [py<38]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  ignore_run_exports:
+    - python
+    - libstdcxx-ng
```

Comment:
why the `ignore_run_exports`?

Reply:
These generate overdepending ERRORs:

```
  ERROR (pymongo): dso library package defaults/linux-aarch64::libstdcxx-ng==11.2.0=h1234567_1 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
```

Reply:
You can just remove `{{ compiler('cxx') }}` from build. I only see setup.py compiling C code.
And python in ignore run_exports doesn't make sense to me. 
Also overdepending gives warnings on CI. (This was by choice given ignore_run_exports were sometimes added unnecessarily, and that missing a dependency is worse than declaring an extra one.)

Reply:
Re: `cxx`; yes you're right -- if a compiler went in the recipe previously I don't question it! :)

Re: `python`; interesting, it is a warning for python but not an error.  I wonder if I conflated the two...